src/main/java/dk/cngroup/lentils/service/ProgressService.java

```diff
@@ -0,0 +1,47 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.Progress;
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.repository.ProgressRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.temporal.TemporalAmount;
+import java.util.List;
+
+@Service
+public class ProgressService {
+
+    ProgressRepository progressRepository;
+
+    @Autowired
+    public ProgressService(ProgressRepository progressRepository) {
+        this.progressRepository = progressRepository;
+    }
+
+    public ProgressService() {
```

Comment:
Pokud používáš injection, tady ten prázdný konstruktor už bych nedával. Pak by si mohla vytvořit tu službu ve špatném stavu.